templates/includes/header.html

```diff
@@ -10,27 +10,10 @@
         {% if link.section %}
         {% set href = "/" ~ link.section ~ "/" %}
         {% set section = get_section(path=link.section ~ "/_index.md") %}
-        <details {{ util::current_open(if_path_starts_with=href) }} class="section-wrap mobile">
```

Comment:
You removed the use of `current_open`, maybe you should turn it into `current_checked` instead?

Reply:
This one actually always returned false, because `current_path` is not defined in `header.html`